skops/io/_sklearn.py

```diff
@@ -12,35 +12,41 @@
     from sklearn.covariance._graph_lasso import _DictWithDeprecatedKeys
 except ImportError:
     _DictWithDeprecatedKeys = None
-from sklearn.linear_model._sgd_fast import (
-    EpsilonInsensitive,
-    Hinge,
-    Huber,
-    Log,
-    LossFunction,
-    ModifiedHuber,
-    SquaredEpsilonInsensitive,
-    SquaredHinge,
-    SquaredLoss,
-)
+try:
+    # TODO: remove once support for sklearn<1.6 is dropped.
+    from sklearn.linear_model._sgd_fast import (
+        EpsilonInsensitive,
+        Hinge,
+        Huber,
+        Log,
+        LossFunction,
+        ModifiedHuber,
+        SquaredEpsilonInsensitive,
+        SquaredHinge,
+        SquaredLoss,
+    )
+
+    ALLOWED_SGD_LOSSES = {
+        ModifiedHuber,
+        Hinge,
+        SquaredHinge,
+        Log,
+        SquaredLoss,
+        Huber,
+        EpsilonInsensitive,
+        SquaredEpsilonInsensitive,
+    }
+    SKLEARN_GT_15 = False
```

Comment:
we don't really care about the version, I think this is more apt:

```suggestion
    SKLEARN_SGD_LOSS = True
```